tests/acceptance/features/coreApiWebdavPreviews/previewsAutoAdustedSizing.feature

```diff
@@ -21,16 +21,16 @@ Feature: sizing of previews of files downloaded through the webdav API
       | request-width | request-height | return-width | return-height | dav-path-version |
       | 1             | 1              | 16           | 16            | old              |
       | 32            | 32             | 32           | 32            | old              |
-      | 1024          | 1024           | 640          | 480           | old              |
+      | 1024          | 1024           | 640          | 640           | old              |
       | 1             | 1024           | 16           | 16            | old              |
-      | 1024          | 1              | 640          | 480           | old              |
+      | 1024          | 1              | 640          | 640           | old              |
       | 1             | 1              | 16           | 16            | new              |
       | 32            | 32             | 32           | 32            | new              |
-      | 1024          | 1024           | 640          | 480           | new              |
+      | 1024          | 1024           | 640          | 640           | new              |
       | 1             | 1024           | 16           | 16            | new              |
-      | 1024          | 1              | 640          | 480           | new              |
+      | 1024          | 1              | 640          | 640           | new              |
       | 1             | 1              | 16           | 16            | spaces           |
       | 32            | 32             | 32           | 32            | spaces           |
-      | 1024          | 1024           | 640          | 480           | spaces           |
+      | 1024          | 1024           | 640          | 640           | spaces           |
       | 1             | 1024           | 16           | 16            | spaces           |
-      | 1024          | 1              | 640          | 480           | spaces           |
+      | 1024          | 1              | 640          | 640           | spaces           |
```

Comment:
by default vips is not enabled, we need to think of running test locally with vips disabled.
May be we should check using either .. or ..
```suggestion
      | 1024          | 1              | 640          | 480,640           | spaces           |
```